### ISSUE TITLE
Toggling "Show all files" in solution explorer replaces project in editor

### ISSUE THREAD

**Issue by dsplaisted** [dotnet/roslyn-project-system#922]
- Create a new .NET Core Console app
- Right click on the project, chose Edit .csproj file
- Click "Show all files" button in solution explorer

Expected: Solution explorer shows all files
Actual: Solution explorer shows all files, but contents of the editor with the project file in it are replaced with:

```xml
<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup>
    <ShowAllFiles>true</ShowAllFiles>
  </PropertyGroup>
</Project>
```

@333fred

**Comment by 333fred** [comment]
This is actually a side-effect of the same bug that https://github.com/dotnet/roslyn-project-system/pull/923 addresses.

**Comment by 333fred** [comment]
Fixed in #923.
